### DISCUSSION TITLE
Add option to disable budgie-wm (to use i3 for example)

### DISCUSSION THREAD

**Discussion by antoniotrkdz** [BuddiesOfBudgie/budgie-desktop#340] [Ideas]
Hi,
I am sort of experimenting with budgie and, as the title says, It would be nice to add a way to remove budgie-wm from the list of required components.
it would be nice to experiment with the different window managers out there (ie tiling window managers).
Both Xfce and Mate provide this option through gsettings.
I am curious to know if it is a lot of work to implement this feature?
Could it be so simple as to provide a build without budgie-wm [here](https://github.com/solus-project/budgie-desktop/blob/979c151f51c40952d3fc7cb26d9e3ad34f640ac3/src/session/meson.build#L44)?

Thank you in advance for your attention.

**Comment by JoshStrobl** [comment]
There are no plans to supports Budgie being decoupled from its window manager at this time.
